Sources/SwiftJWT/BlueRSA.swift

```diff
@@ -20,7 +20,8 @@ import LoggerAPI
 
 import Foundation
 
-class BlueRSA: EncryptionAlgorithm {
+class BlueRSA: SignerAlgorithm, VerifierAlgorithm {
```

Comment:
Does BlueRSA now provide enough Linux support that we could use this code on Linux and get rid of the Linux-specific `RSA` type?